### ISSUE TITLE
Building Serenity on Arch Linux

### ISSUE THREAD

**Issue by xyproto** [SerenityOS/serenity#74]
Hi,

I just git cloned the latest `master`, entered the `Toolchain` directory and went:

`./BuildIt.sh`

The last lines of the output were:

```sh
CXX sys/socket.cpp
CXX sys/wait.cpp
CXX sys/uio.cpp
CXX poll.cpp
CXX locale.cpp
locale.cpp:11:1: warning: ISO C++ forbids converting a string constant to 'char*' [-Wwrite-strings]
};
^
locale.cpp:11:1: warning: ISO C++ forbids converting a string constant to 'char*' [-Wwrite-strings]
locale.cpp:11:1: warning: ISO C++ forbids converting a string constant to 'char*' [-Wwrite-strings]
CXX arpa/inet.cpp
CXX netdb.cpp
CXX sched.cpp
AS setjmp.ao
AS crti.ao
AS crtn.ao
LIB libc.a
mkdir -p ../Root/usr/include
mkdir -p ../Root/usr/lib
# Copy headers
rsync -r -a --include '*/' --include '*.h' --exclude '*' . ../Root/usr/include
# Install the library
cp libc.a ../Root/usr/lib
cp crt0.o ../Root/usr/lib/
cp: klarte ikke å hente informasjon om 'crt0.o': Ingen slik fil eller filkatalog
make: *** [Makefile:86: install] Error 1
make: Leaving directory '/home/afr/clones/serenity/LibC'
```

"Ingen slik fil eller filkatalog" means "No such file or directory" in Norwegian.

Using `export LC_ALL=C` or `export LC_ALL=en_US.UTF-8` in the build script will produce error messages in English instead of the local preference.

I am unable to see, at a glance, why it can't find `crt0.o`.

This is with `gcc 8.3.0` on 64-bit x86.

Thanks for creating an alternative operating system! I'm looking forward to trying it out one day.

**Comment by awesomekling** [comment]
I just pushed a fix for this in 709893364df1241da94c6396a7ab1440aab7593d
It was a Makefile target dependency problem :)

Thanks for reporting it! Hope the rest of the build process goes smoothly.

**Comment by xyproto** [comment]
Now it works, thanks. :+1:
